elements/bulbs-header/bulbs-header.js

```diff
@@ -0,0 +1,65 @@
+import { registerElement, BulbsHTMLElement } from 'bulbs-elements/register';
+import './bulbs-header-masthead';
+import './bulbs-header-responsive-nav';
+import './bulbs-header.scss';
+
+import { InViewMonitor } from 'bulbs-elements/util';
+
+class BulbsHeader extends BulbsHTMLElement {
+  createdCallback () {
+    this.mastheads = this.getElementsByTagName('bulbs-header-masthead');
+    this.responsiveNavs = this.getElementsByTagName('bulbs-header-responsive-nav');
```

Comment:
The use of `getElementsByTagName` and the `[0]` lookup is defensive. `getElementsByTagName` returns a live collection that updates as the dom updates.

Always looking at the head of that collection means you can break the reference to the actual internal elements without breaking the element as a whole.

But it could be too defensive, this would also work:

```js
this.masthead = this.querySelector('bulbs-header-masthead');
this.responsiveNav = this.querySelector('bulbs-header-responsive-nav');
```

And then delete the getters.